web/conf.py

```diff
@@ -17,7 +17,7 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
 # build docs using nltk from the upper dir, not the installed version
-sys.path.insert(0, os.path.abspath('..'))
```

Comment:
it perhaps looks like you have some black changes in this diff? these are all totally okay though!

Reply:
Indeed. I mentioned it briefly in *Other notes*, but 8 files were affected by the whitespace trimmer, and 8 files were reformatted with `black`.